catalog/dags/common/loader/provider_details.py

```diff
@@ -12,6 +12,7 @@
 
 
 # Default provider names
+AUCKLAND_MUSEUM_IMAGE_PROVIDER = "aucklandmuseum"
```

Comment:
Unfortunately it's not realistic to have diacritical marks (macron) on these slugs, but we can still have both. I left out "war memorial" from the colonial name because it's less important and the institution itself doesn't always include it.

```suggestion
AUCKLAND_MUSEUM_IMAGE_PROVIDER = "aucklandmuseum_tamakipaengahira"
```

Reply:
This is good to hear. The changes are made.